### PR TITLE
Insufficient funds screen

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[New machine screen for 'insufficient funds' error, in lieu of 'We got this'](https://trello.com/c/wjkyZMSO/839-new-machine-screen-for-insufficient-funds-error-in-lieu-of-we-got-this)

File: ui/src/app.js [hunk 1]
```diff
@@ -198,7 +198,8 @@ function processData (data) {
       break
     case 'balanceLow':
     case 'insufficientFunds':
-      setState('limit_reached')
+      // setState('limit_reached')
```
Comment (josepfo) [inline ui/src/app.js:201]: Unnecessary comment

**Review by josepfo — CHANGES_REQUESTED** [review]
Just a comment. The qr image seems too small compared to the spec and that causes the underlying content to be unaligned with left side. Can you take a look at that?

**Review by josepfo — APPROVED** [review]

File: lib/brain.js [hunk 2]
```diff
@@ -3076,10 +3076,11 @@ Brain.prototype._sendCoinsError = function _sendCoinsError (err) {
   const self = this
   if (err.statusCode === INSUFFICIENT_FUNDS_CODE) {
     setTimeout(function () { self._idle() }, self.config.insufficientFundsTimeout)
-    return this._transitionState('insufficientFunds')
+    this._transitionState('insufficientFunds', withdrawFailureRec)
+  } else {
+    this._transitionState('withdrawFailure', withdrawFailureRec)
   }
 
-  this._transitionState('withdrawFailure', withdrawFailureRec)
```
Comment (RafaelTaranto) [inline lib/brain.js:3082]: Why was this change needed?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/brain.js (hunk 2 above)
Comment (chaotixkilla) [reply lib/brain.js]: Probably some leftover from earlier development. Fixed

**Review by RafaelTaranto — APPROVED** [review]
